net/chrony/src/opnsense/service/conf/actions.d/actions_chrony.conf

```diff
@@ -21,3 +21,27 @@ command:/usr/local/etc/rc.d/chronyd status;exit 0
 parameters:
 type:script_output
 message:request chrony status
+
+[chronysources]
```

Comment:
I’m unsure we should use chrony prefix here, configctl chrony chronysources looks weird. But not a blocker.